### PR TITLE
Cross platform support to run yarn test:edge for Microsoft Edge dev tools extension

### PR DESCRIPTION
**Pull request by hmohamme**
## Summary

Follow up PR for [this comment](https://github.com/facebook/react/pull/18041#pullrequestreview-360539853). 

Changes:
Added `open` and `os-name` dependencies.
Removed `windows-edge` dependency.
Modified code in `packages/react-devtools-extensions/edge/test.js` to open Microsoft Edge on Mac and Windows devices.

## Test Plan
1. Run `yarn build:edge:dev`
2. Follow the instructions on console to load the extension.
3. Run `yarn run test:edge`

### CODE REVIEW

**Comment by hmohamme** [comment]
I was having issues with adding packages with yarn. I think [because of this](https://github.com/yarnpkg/yarn/issues/3973#issuecomment-585465518). I added the packages using `npm` and tried to generate `yarn.lock` file using `yarn install`. I apologize in advance if the yarn.lock file is messed up. Let me know if I need to revert it.

**Comment by bvaughn** [comment]
Thanks for following up on this :)

**Comment by hmohamme** [comment]
I have not yet found a way to side load extensions using command line switches for Edge. I have tried `--load-extension="<path">`  or `--extension=<"path">`, but they doesn't seem to work.

That is the reason I have changed the instructions on console to manually load the extension for the first time.
image

**Comment by bvaughn** [comment]
Ah, hm. I see. I'm concerned that this difference between the Chrome, Firefox, and Edge `test` targets might cause people to think they've verified something when they haven't yet

**Comment by hmohamme** [comment]
I understand. I have submitted a bug on the edge feedback forum. I will wait for their response. Thank you for looking at the PR.

**Comment by bvaughn** [comment]
No problem :) Thanks for the follow up

**Comment by hmohamme** [comment]
Turns out that I need to have[ edge closed before loading](http://www.chromium.org/developers/how-tos/run-chromium-with-flags) the extensions via command line.

Added code to automatically load the extension. Ready for review again. :sweat_smile:

File: packages/react-devtools-extensions/edge/test.js [hunk 1]
```diff
@@ -2,17 +2,28 @@
...
 
-const edge = require('windows-edge');
-
+const open = require('open');
+const os = require('os');
+const osName = require('os-name');
 const START_URL = 'https://facebook.github.io/react/';
+const {resolve} = require('path');
+
+const EXTENSION_PATH = resolve('./chrome/build/unpacked');
```
Comment (bvaughn) [inline packages/react-devtools-extensions/edge/test.js]: Whoops. This is the wrong path 😄

**Comment by hmohamme** [comment]
Just saw this. Sorry for the typo and thank you so much for fixing it. :raised_hands:

**Pull request merged**